### PR TITLE
Clean up test database before each new Cypress test

### PR DESCRIPTION
**Pull request by valscion**
This PR is a follow-up to #32

---

This PR runs [`database_cleaner`](https://github.com/DatabaseCleaner/database_cleaner) gem's database cleanup before each Cypress test is ran so that all tests run from a clean slate.

We use a `beforeEach()` hook to do that as that's what Cypress suggests: https://on.cypress.io/best-practices#Using-after-or-afterEach-hooks

### CODE REVIEW

**Comment by jraunio** [comment]
👍

**Comment by JonasThers** [comment]
Just so I'm sure: [Here](https://github.com/venuu/venuu/blob/master/cypress/e2e/venueInquiry/email-marketing-consent.spec.js#L52) we have a case where we access the db to make sure that we the expected data has actually been sent. But every time we run the test, we only get the same line of data, because that's all there is the database when we run the test. This is what this `database_cleaner`does, right? I mean, makes sure that we don't save data from previously ran test?

File: cypress/e2e/root-page.cy.js [hunk 1]
```diff
@@ -4,15 +4,7 @@ describe('Root page', () => {
...
     // Fill in the signup form and submit it.
-    //
-    // We need to use something random for the email as test database is not
-    // cleaned up between tests and emails must be unique!
-    //
-    // TODO: Clean up Rails test database before each test so tests are isolated!
-    //       It's a good practice on doing the cleanup in `beforeEach`:
-    //       https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks
-    //
-    cy.findByRole('textbox', { name: 'Email'}).type(`user-${Math.round(Math.random() * 1_000_000)}@example.com`)
```
Comment (JonasThers) [inline cypress/e2e/root-page.cy.js:15]: So now when the database gets cleaned, we don't have to create a new user for every test? We can just same the credentials every time?

**Comment by JonasThers** [comment]
If my comments are correctly understood, then everything else seems clear 👍

**Comment by valscion** [comment]
Yeah you understood everything correctly :+1:

We do indeed now have an empty database at the start of every test thanks to `database_cleaner`.

**Pull request merged**